client/src/core/services/parse-map.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+      providedIn:'root',
+})
+export class ParseMapService {
+    terrain: any = {
+        name: 'Terrain 1',
+        height: 30,
+        width: 50,
+        obstacles: [],
+        trees: [],
+    };
+
+        constructor() { }
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5943029)